### PR TITLE
Add busybox network configuration script

### PR DESCRIPTION
**Pull request by igoropaniuk**
`Signed-off-by: Igor Opaniuk <igor.opaniuk@linaro.org>`

### CODE REVIEW

File: filelist.txt [hunk 1]
```diff
@@ -48,6 +48,8 @@ file /etc/gshadow etc/gshadow 644 0 0
 file /etc/fstab etc/fstab 644 0 0
 file /etc/hosts etc/hosts 644 0 0
 slink /etc/mtab /proc/mounts 755 0 0
+dir /etc/udhcp 755 0 0
+file /etc/udhcp/simple.script stage/etc/udhcp/simple.script 755 0 0
```
Comment (jockebech) [inline filelist.txt:53]: Should it really say **stage**/etc/udhcp/simple.script ... ? Everything else here refers directly to `etc/foo/bar`?

Comment (igoropaniuk) [reply filelist.txt:53]: Another example https://github.com/linaro-swg/gen_rootfs/blob/master/filelist.txt#L2
`file /bin/busybox stage/bin/busybox 755 0 0`
All stuff, that were created during build are put into `stage`(`stage/etc` and etc.) dir. `etc` dir in the root of `gen_rootfs` just contains static stuff

Comment (jockebech) [reply filelist.txt:53]: Ok, I'll rest my case, you've convinced me.

**Comment by jockebech** [comment]
`Reviewed-by: Joakim Bech <joakim.bech@linaro.org>`

**Comment by igoropaniuk** [comment]
Applied tags & rebased

**Comment by igoropaniuk** [comment]
I think I've addressed all comments
@jforissier could you please merge this PR?

**Pull request merged**